features/api-resource-mgt/org.wso2.carbon.identity.api.resource.mgt.server.feature/resources/api-resource-collection.xml

```diff
@@ -823,9 +1626,29 @@
         </Scopes>
     </APIResourceCollection>
     <APIResourceCollection name="org_gettingStarted" displayName="Home Page" type="organization">
-        <Scopes>
+       <Scopes>
```

Comment:
We can have a comment for the previous scope collections saying keeping for backward compatibility.

Reply:
+1